test/onnx/onnx_test.cpp

```diff
@@ -4904,6 +4904,118 @@ TEST_CASE(qlinearadd_test)
     EXPECT(p.sort() == prog.sort());
 }
 
+TEST_CASE(qlinearmatmul_1D_test)
```

Comment:
Missing 3D parser test. If it looks similar to the 2D test, you may be able to make a generic function that can return the migraphx program adjusted for the inputs' dimensions.

Reply:
It is the 1D test that is a bit special.. hence there are two tests: the other one is for 2D. It works (similarly) for anything beyond 2D. I am not sure what generic function are you referring to here in the unit-test, please? 

The generic function is parse_qlinearmatmul.cpp :-)

Reply:
I see. So the 3D onnx file was created for verify_onnx and wasn't going to add test any additional cases in the onnx parser.